.condarc.yaml

```diff
@@ -0,0 +1,6 @@
+channels:
+  - defaults
+  - bioconda
+  - conda-forge
```

Comment:
These are the channels you need during setup, right?

Reply:
Yes, conda-forge and bioconda channels host the packages autometa requires